### ISSUE TITLE
WAF Metrics 

### ISSUE THREAD

**Issue by pszeto** [k8sgateway/k8sgateway#7554]
### Version

1.12.x (latest stable)

### Is your feature request related to a problem? Please describe.

Currently when WAF is enabled on the gateway, activity is written to logs: https://docs.solo.io/gloo-edge/latest/guides/security/waf/#audit-logging

### Describe the solution you'd like

It would be nice to have metrics on request blocked by WAF rules.

**Comment by edubonifs** [comment]
Hi,
T-Systems is also asking for this
